packages/@aws-cdk/cx-api/lib/context/vpc.ts

```diff
@@ -0,0 +1,29 @@
+export const VPC_PROVIDER = 'vpc-provider';
+
+export interface VpcContextQuery {
+  /**
+   * Query account
+   */
+  account?: string;
+
+  /**
+   * Query region
+   */
+  region?: string;
+
```

Comment:
I would put this under `filters: any[]` to avoid the need to define this in 100% places.